mypy.ini

```
@@ -26,3 +26,5 @@ pretty = True
 
 [mypy-torchvision.*]
 ignore_missing_imports = True
+[mypy-PIL.*]
```

Add blank line in between

```suggestion

[mypy-PIL.*]
```